src/components/views/dialogs/TextInputDialog.js

```diff
@@ -40,6 +42,7 @@ export default class TextInputDialog extends React.Component {
         title: "",
         value: "",
         description: "",
+        busyMessage: "Loading...",
```

Comment:
```suggestion
        busyMessage: _td("Loading..."),
```

for the i18n script to work